tests/Doctrine/Tests/ORM/Cache/Persister/Entity/AbstractEntityPersisterTest.php

```diff
@@ -193,11 +199,12 @@ public function testInvokeDelete(): void
 
         $this->entityPersister->expects(self::once())
             ->method('delete')
-            ->with(self::equalTo($entity));
+            ->with(self::identicalTo($entity))
+            ->willReturn(true);
 
         $this->em->getUnitOfWork()->registerManaged($entity, ['id' => 1], ['id' => 1, 'name' => 'Foo']);
 
-        self::assertNull($persister->delete($entity));
```

Comment:
The method declares `bool` as return type but we assert `null`. We're basically testing a broken mock configuration.